packages/storage/jest.config.ts

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import baseDefaultConfig from '@fuel-ui/test-utils/config';
+import type { JestConfigWithTsJest } from 'ts-jest';
+import { defaultsESM as tsjPreset } from 'ts-jest/presets';
+
+import pkg from './package.json';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const { globals, preset, ...baseConfig } = baseDefaultConfig;
```

Comment:
There's probably a way to remove the globals and preset properties on baseDefaultConfig through some sort of ts magic.  That way we could remove the eslint-disable, but I can't think of the code off the top of my head so it probably isn't worth the extra work.

Reply:
we should not focus on reviewing/changing new packages: `storage` and `store`. reasons

- they're here only temporarily, and will be replaced later when FRO-24 gets done
- we don't wanna start having differences from wallet implementation. because the code there is already in master, stable and reviewed. if we start changing we open space for bugs in portal side